src/Microsoft.Identity.Abstractions/ManagedIdentity/ManagedIdentityType.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Abstractions
+{
+    /// <summary>
+    /// Used by <see cref="ManagedIdentityOptions"/> to specify the type of managed identity to use.<br/><br/>
+    /// See <see href="https://aka.ms/Entra/ManagedIdentityOverview"/> for more details.
+    /// </summary>
+    public enum ManagedIdentityType
+    {
+        /// <summary>
+        /// The default value, indicating the managed identity to use is the one configured for the Azure resource on which the
```

Comment:
EDAP or Azure resource? @jmprieur ?

Reply:
yes, managed identity are configured on Azure resources (computes) in general